### DISCUSSION TITLE
Now Timing in with INIT and Stat Analysis

### DISCUSSION THREAD

**Discussion by CPKalb** [dtcenter/METplus#1668] [Configuration] [answered]
Hi:

Is there something specific that needs to be done to use the now keyword with an INIT_BEG 

in the stat analysis wrapper?  I have my timing information set as such:
INIT_BEG = {now?fmt=%Y%m%d?shift=-172800}
INIT_END = {now?fmt=%Y%m%d?shift=-172800}

However, this produces as error as shown in the attached file.  The script runs fine if I set the timing information as:
INIT_BEG = 20220615
INIT_END = 20220615

Thanks,
Christina

[metplus_log_20220617210126.txt](https://github.com/dtcenter/METplus/files/8931373/metplus_log_20220617210126.txt)

**Comment by JohnHalleyGotway** [comment, marked as answer]
@CPKalb thanks for pointing this out. I independently tested this myself and confirmed that neither `now` or `today` works in the METplus StatAnalysis conf files, as described in [this section](https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html?highlight=now#now-and-today) of the user's guide. I wrote this up as a [bugfix issue](https://github.com/dtcenter/METplus/issues/1669) and recommend that we move further discussion on this from this discussion topic over to commenting directly on that issue.

If you agree, please mark this discussion as being answered and then lock it to prevent future posts.
